voltsdk/src/main/java/com/voltmoney/voltsdk/VoltWebViewActivity.kt

```diff
@@ -585,12 +592,20 @@ class VoltWebViewActivity : AppCompatActivity() {
                 return true
             } else if (checkURLMatchesFromListArray(url, urlOpenInCustomTab)) {
                 Log.d("REACHED THERE IN THE", "")
-               var  customTabsIntent = CustomTabsIntent.Builder().setInitialActivityHeightPx(400,
+                var customTabsIntent = CustomTabsIntent.Builder().setInitialActivityHeightPx(
+                    400,
                     CustomTabsIntent.ACTIVITY_HEIGHT_FIXED
                 )
-                customTabsIntent!!.setToolbarColor(ContextCompat.getColor(this@VoltWebViewActivity, androidx.appcompat.R.color.material_blue_grey_800));
+                customTabsIntent!!.setToolbarColor(
+                    ContextCompat.getColor(
+                        this@VoltWebViewActivity,
+                        androidx.appcompat.R.color.material_blue_grey_800
+                    )
+                );
                 openCustomTab(this@VoltWebViewActivity, customTabsIntent!!.build(), Uri.parse(url));
 
+            } else if (url?.contains("http://google.com/exitAndroidSDK")) {
```

Comment:
Why there is google.com here

Reply:
This is just a dummy url with appended values?

Reply:
ok